stable/distribution/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: distribution
 home: https://jfrog.com/platform/
 description: A Helm chart for JFrog Distribution
-version: 7.1.0
+version: 7.2.0
```

Comment:
@rahulsadanandan Distribution 2.5.1 was released , can you bump that version

Reply:
okay.

Reply:
Done